app/views/spree/admin/adjustments/index.html.haml

```diff
@@ -7,8 +7,12 @@
 
 - content_for :page_actions do
   %li= button_link_to t(:new_adjustment), new_admin_order_adjustment_url(@order), :icon => 'icon-plus'
+  %li.links-dropdown#links-dropdown{ links: order_links(@order).to_json }
   %li= button_link_to t(:back_to_orders_list), admin_orders_path, :icon => 'icon-arrow-left'
 
 = render :partial => 'adjustments_table'
 
 = button_link_to t(:continue), admin_order_payments_url(@order), :icon => 'icon-arrow-right'
+
+:coffee
+  angular.bootstrap(document.getElementById("links-dropdown"),['admin.dropdown'])
```

Comment:
Hey, I thought that was weird at first glance, but since I don't understand Angular a lot, I preferred not to change this part. Can you help me move it to a directive or a controller? I checked other controllers as an example, but I was not able to make it work.

Reply:
ah, I looked at the code again and this is already a directive!
linksDropdown.js.coffee

I see this bootstrap thing comes from a defaced spree view (history).
I wonder if we even need this now. Have you tried removing the line and see if the directive still works?
The class .links-dropdown should already make the directive work.

Reply:
You are correct, it works!

Reply:
> Nice one!
> 
> Yes, view specs would be ideal for this! We already have some view specs for the orders page, should be easy to verify the actions menu is there. I am not so sure how easy it will be to assert the individual actions are there or to open the actions menu and click on one. Can you pls give it a try?

@luisramos0 I tried and I don't think I will be able to test the action with view tests. If we want to test the action too I think I should create a feature spec.

Reply:
yeah, as you wish, I'd maybe just have the view spec test the menu is present.

Reply:
To be clear, a couple of view specs verifying the actions menu appears in all the pages is good enough imo.

Reply:
Hey @luisramos0, you were perfectly clear! Last week was busy, but I will try to create those specs this week.

Reply:
I was not able to create view specs. I think it is something related to those buttons being generated with the coffee.
In the output of the failed test, I noticed this: `function() { angular.bootstrap(document.getElementById(\"links-dropdown\"), ['admin.dropdown']); }).call(this)`. It looks like the buttons are not being rendered on the specs.

Reply:
I pushed the spec if you want to see the test output. I thought it was too big to copy and past here.